.github/workflows/integration_tests.yml

```diff
@@ -52,47 +52,13 @@ jobs:
           fi
 
       - name: Start test Docker database
-        working-directory: integration_tests
+        working-directory: integration_tests/mock
```

Comment:
what do you think of leaving the docker-compose.yml on the top level integration_tests?